Roys/app/src/main/java/com/dwg_karrier/roys/ContentView.java

```diff
@@ -0,0 +1,34 @@
+package com.dwg_karrier.roys;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.text.method.ScrollingMovementMethod;
+import android.util.Log;
+import android.widget.TextView;
+
+public class ContentView extends AppCompatActivity {
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.content);
+    try {
+      /* TODO
+      * bring user-clicked URL
+      * merge with soyee's code
+      */
+      // yet, write exact pageUrl
+      String pageUrl = "https://mercury.postlight.com/parser?url=http://www.bloter.net/archives/265787";
+      Crawler crawler = new Crawler(pageUrl);
+      // show title and content in one page
+      // It's better to scroll down in concatenated version
+      String title = crawler.getTitle();
+      String content = crawler.getContent();
+      TextView contentView = (TextView) findViewById(R.id.contentView);
+      contentView.setText(title+"\n\n"+content);

```

Comment:
nit: https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
Did the Google Java convention plugin applied? I believe this type of convention should be applied automatically.